### ISSUE TITLE
Unable to run immortal with yml config in docker

### ISSUE THREAD

**Issue by alexzaytsev-newsroomly** [immortal/immortal#23]
I am trying to create long-running services with .net core and Docker, and i want them supervised with immortal.

Here's my dockerfile:

```
FROM microsoft/dotnet:2.0-runtime
    ARG source
    WORKDIR /app
    COPY ${source:-obj/Docker/publish} .
    WORKDIR /immortal
    RUN  apt-get update \
      && apt-get install -y wget \
      && rm -rf /var/lib/apt/lists/*
    RUN wget https://github.com/immortal/immortal/releases/download/0.16.0/immortal_0.16.0_linux_amd64.tar.gz
    RUN tar -xzvf immortal_0.16.0_linux_amd64.tar.gz
    WORKDIR /app
    ENTRYPOINT ["/immortal/immortal_0.16.0_linux_amd64/immortal", "-c /app/coresandbox.yml"]
```

(i am using VS Docker tools, so the publishing to `/obj/Docker/publish` is handled outside of this docker file)

If i do ```docker run coresandbox``` i get this output:

  ```  
flag provided but not defined: -c /app/coresandbox.yml
    Usage: /immortal/immortal_0.16.0_linux_amd64/immortal [-v] [-ctl dir] [-d dir] [-e dir] [-f pidfile] [-l logfile] [-logger logger] [-p child_pidfile] [-P supervisor_pidfile] [-u use
    
       command
            The command with arguments if any, to supervise
    
      -c run.yml
            run.yml configuration file
      -ctl /var/run/immortal/<service>
            Create supervise directory /var/run/immortal/<service>
      -d dir
            Change to dir before starting the command
      -e dir
            Set environment variables specified by files in the dir
      -f pidfile
            Follow PID in pidfile
      -l logfile
            Write stdout/stderr to logfile
      -logger command
            A command to pipe stdout/stderr to stdin
      -P pidfile
            Path to write the supervisor pidfile
      -p pidfile
            Path to write the child pidfile
      -s seconds
            seconds to wait before starting
      -u user
            Execute command on behalf user
      -v    Print version
```

However, if i use bash interactively in the image, i can start immortal:

```docker run --entrypoint /bin/bash -i -t coresandbox```

```/immortal/immortal_0.16.0_linux_amd64/immortal -c /app/coresandbox.yml```

   ``` 2017/09/13 04:47:01 ⭕  86
    2017/09/13 04:47:02 PID 90 terminated, exit status 0 [100ms user  50ms sys  156.8319ms up]
    2017/09/13 04:47:03 PID 103 terminated, exit status 0 [110ms user  40ms sys  154.0168ms up]
```

The contents of my ```coresandbox.yml``` are correct yaml, and correct config for immortal

**Comment by alexzaytsev-newsroomly** [comment]
The problem was with my docker entrypoint, using the direct format ```ENTRYPOINT /immortal/immortal_0.16.0_linux_amd64/immortal -c /app/coresandbox.yml``` worked

**Comment by nbari** [comment]
Hi @alexzaytsev-newsroomly nice that is working, if you find a problem or need help with something please don't hesitate on creating another issue.
